vscode-dotnet-runtime-library/src/Acquisition/DotnetCoreAcquisitionWorker.ts

```diff
@@ -113,6 +115,11 @@ export class DotnetCoreAcquisitionWorker implements IDotnetCoreAcquisitionWorker
         const dotnetInstallDir = this.context.installDirectoryProvider.getInstallDir(version);
         const dotnetPath = path.join(dotnetInstallDir, this.dotnetExecutable);
 
+        if (fs.existsSync(dotnetPath) && installedVersions.length === 0) {
+            // A local install already exists, add it to our managed installs
```

Comment:
I'd expand this comment because it's a little ambiguous but inherently dependent on the idea that an external asset (the edu bundle) has done something that we can react to. It's not that just any pre-install was detected (aka we're not looking for Admin installs) it's that a "local install" was detected as in someone copy and pasted some content into the VSCode folder.